src/FirstSteps/RPG/Game.cs

```diff
@@ -78,5 +53,39 @@ public static void DisplayAllRaces()
                 Console.WriteLine($"{i + 1}. {races[i]}");
             }
         }
+        public static void GetMenu()
```

Comment:
Try to choose a better name because `Get` in the method name can suggest that it returns the menu. Maybe the name could relate to the process of choosing/making player decisions.